images/inference-services/seldon-core/run.sh

```diff
@@ -50,7 +50,7 @@ envsubst < /root/template.sh | kubectl apply -f -
 
 # rollout fails if the object does not exist yet, so we need to wait until it is created
 count=0
-until [[ -n "$(kubectl get deploy -l seldon-deployment-id=${sd} -n ${FUSEML_ENV_WORKFLOW_NAMESPACE} 2>/dev/null)" ]]; do
+until kubectl get deploy -l seldon-deployment-id=${sd} -n ${FUSEML_ENV_WORKFLOW_NAMESPACE} 2>/dev/null; do
```

Comment:
I do not think this is right; I had this before but this checks for exit codel however if you search with `-l`, you get exit code 0 even if the resource does not exist

Reply:
Apologies, you're totally right, this shouldn't even be part of this PR, it was an omission on my part.